addons/hr_holidays/models/hr_leave.py

```diff
@@ -515,6 +516,11 @@ def _compute_number_of_days(self):
             else:
                 holiday.number_of_days = 0
 
+    @api.depends('number_of_days')
+    def _compute_last_several_days(self):
+        for holiday in self:
+            holiday.last_several_days = holiday.number_of_days > 1
```

Comment:
Is that condition covering every case? I feel like some part-time worker could take an hour-based time-off for which the number of days would be resulting in being less than one :thinking: Though I'm not sure how much of a corner case it would be and honestly can't come up with a definitive solution... Maybe comparing `date_from.date()` and `date_to.date()` ?

Reply:
AFAIK `number_of_days` takes into account the work schedule of the employee. Check https://github.com/odoo/odoo/blob/29b0bb645c17afe1d2462a73c50ae414374f2ffb/addons/hr_holidays/models/hr_leave.py#L758-L766